Resources/public/js/views/ezsearch-searchview.js

```diff
@@ -12,8 +12,16 @@ YUI.add('ezsearch-searchview', function (Y) {
             this.after('userSearchStringChange', function() {
                 this.fire('searchRequest', {
                     searchString: this.get('userSearchString'),
+                    limit: 10,
```

Comment:
you should use `loadMoreAddingNumber` attribute value instead of hardcoding 10 here